### DISCUSSION TITLE
Publii Website: Download Theme in Firefox doesn't work with AdBlocker enabled

### DISCUSSION THREAD

**Discussion by Jugibur** [GetPublii/Publii#1444] [Bug Report]
### Operating system

macOS Ventura

### Publii version

(not relevant)

### Issue type

Free theme

### Bug description

Hi,
I just wanted to try out Publii and realized that a theme is not downloading in Firefox with "AdBlock Ultimate" enabled.
When I try it in Safari, it works.
For example this theme:
https://marketplace.getpublii.com/themes/qf-fashion/

I just wanted to share this, as most users don't think of deactivating their adblocker as a test. 
Perhaps add a note or change the download technique.

### Steps to reproduce

Installing "AdBlock Ultimate"
Click on "Download" inside the theme page.

**Comment by bobmitro** [comment]
@Jugibur, thank you for the alert. The issue should now be resolved.
